src/main/drivers/bus_i2c_hal.c

```diff
@@ -40,42 +40,72 @@ static void i2cUnstick(IO_t scl, IO_t sda);
 #define IOCFG_I2C IOCFG_AF_OD
 #endif
 
+#define DEF_I2C1_SCL PB6
```

Comment:
This level of indirection is probably unnecessary, it is possible to use pin names when defining i2cHardwareMap